src/components/print/PrintDirective.js

```diff
@@ -535,10 +532,30 @@ goog.require('ga_time_service');
 
       // Transform layers to literal
       layers.forEach(function(layer) {
-        if (layer.visible && (!layer.timeEnabled ||
-            angular.isDefined(layer.time))) {
+        if (!layer.visible || layer.opacity == 0) {
```

Comment:
technically it's impossible a preview layer is displayed when you push print button.